Bottom: 75e2d689174d4da2592ec915c8e814a833ae63c2
Top:    21a3eb4fa6542dfd936115f7f3d204d736b581e2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 21:49:31 +0100

:(.. SPENT ALL THAT TIME FIXING MODEL ROTATION, AND SEEMS TO HAVE BEEN DELETED!


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 5d4c67e..8807835 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -912,6 +912,8 @@ step_do_export (HID_Attr_Val * options)
         parse_numeric_attribute (element, "PCB::3d_model::ref_dir::Y", &ry);
         parse_numeric_attribute (element, "PCB::3d_model::ref_dir::Z", &rz);
 
+        parse_numeric_attribute (element, "PCB::3d_model::rotation", &rotation);
+
         printf ("Transform (%f, %f, %f), (%f, %f, %f), (%f, %f, %f)\n", ox, oy, oz, ax, ay, az, rx, ry, rz);
 
         model = NULL;
